### ISSUE TITLE
unable to open database file: out of memory

### ISSUE THREAD

**Issue by RaSerge** [uptrace/helm-charts#8]
Hello!
With enabled  default securityContext's (i just uncomment it in values.yaml)

```
  podSecurityContext:
    fsGroup: 2000
  securityContext:
    capabilities:
      drop:
      - ALL
    readOnlyRootFilesystem: true
    runAsNonRoot: true
    runAsUser: 1000
```

we catch error with not writable dir for sqlite:

2023-03-16T19:24:00.352Z	info	uptrace/main.go:111	starting Uptrace...	{"version": "v1.3.1", "config": "/etc/uptrace/uptrace.yml"}
2023-03-16T19:24:00.358Z	error	uptrace/main.go:219	SQLite Ping failed; make sure dir is writable (edit db.dsn YAML option)	{"dsn": "file:uptrace.sqlite3?_pragma=foreign_keys(1)&_pragma=busy_timeout(1000)", "error": "unable to open database file: out of memory (14)"}
main.initSqlite

**Comment by vmihailenco** [comment]
In the next release we will always require PostgreSQL so you can just few a week or so while we preparing the release.

**Comment by vmihailenco** [comment]
This should be fixed in v1.4
